### DISCUSSION TITLE
Next Google Font: Source Sans Pro not looking like it should

### DISCUSSION THREAD

**Discussion by Rounak-stha** [vercel/next.js#49688] [Help]
### Summary

HI, I am using Source_Sans_Pro from next/font/google but it does not display as it should.

![image](https://github.com/vercel/next.js/assets/66557682/b4fc7f68-98f8-4121-a4f2-8b1b89dd2020)

I inspected the class added and it seems like only the fallback version is added.

![image](https://github.com/vercel/next.js/assets/66557682/dbefa79b-7b5c-4dec-bac4-08cb3addecdc)

I inspected again with another font .

![image](https://github.com/vercel/next.js/assets/66557682/62d4246c-b77f-49ba-83b2-30b54531223a)

Can anyone help? what seems to be the problem?

**Comment by fabiel-leon** [comment]
source sans pro has been renamed as Source Sans 3
google fonts no longer provides font sans pro

![image](https://github.com/vercel/next.js/assets/9593126/39da6c74-02cd-4fd2-877a-607c47b9b07a)

https://fonts.google.com/specimen/Source+Sans+3
